api/src/main/java/org/pmiops/workbench/billing/OfflineBillingController.java

```diff
@@ -38,16 +35,6 @@
   @Override
   public ResponseEntity<Void> bufferBillingProjects() {
     billingProjectBufferService.bufferBillingProjects();
-    // TODO(jaycarlton) we're going to need a richer framework for cron reporting,
```

Comment:
This metric was no longer in use now that we have a general operation time metric.